udata/assets.py

```diff
@@ -1,91 +1,6 @@
-import io
-import json
-import os
-import pkg_resources
-
-from flask import current_app, request, url_for
+from flask import current_app, url_for
 from flask_cdn import url_for as cdn_url_for
 
-# Store manifests URLs with the following hierarchy
-# app > filename (without hash) > URL
-_manifests = {}
-
-_registered_manifests = {}  # Here for debug without cache
-
-
-def has_manifest(app, filename='manifest.json'):
-    '''Verify the existance of a JSON assets manifest'''
-    try:
-        return pkg_resources.resource_exists(app, filename)
-    except (ImportError, NotImplementedError):
-        return os.path.isabs(filename) and os.path.exists(filename)
-
-
-def register_manifest(app, filename='manifest.json'):
```

Comment:
Some manifest functions were probably used in plugins. Ex: [register_manifest in udata-tabular-preview](https://github.com/opendatateam/udata-tabular-preview/blob/ef67aa5fd77819fb1322958ceb23825aa58d8c06/udata_tabular_preview/views.py#L35).
We should make sure to update existing plugins accordingly (with the version burst solution).